tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/organize_batches_in_pandas_filesystem_datasource.py

```diff
@@ -22,7 +22,6 @@
 context.sources.add_pandas_filesystem(
     name="my_datasource", base_directory=data_directory
 )
-# </snippet>
```

Comment:
eek was this causing problems at all?

Reply:
@Shinnnyshinshin Apparently not!  But at least it will not ever, even if it did!